environment/src/main/java/com/sequenceiq/environment/configuration/telemetry/AccountTelemetryConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.sequenceiq.environment.configuration.telemetry;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import com.sequenceiq.common.api.telemetry.model.AnonymizationRule;
+
+@Configuration
```

Comment:
the main purpose of `@Configuration` is to indicate the class declares `Beans`
I think here we should go with `@Component`